out/startup.js

```diff
@@ -59,3 +59,10 @@ export async function main(ns) {
         await ns.sleep(60000)
     }
 }
+
+export function autocomplete(data, args) {
```

Comment:
You don't need this.  Boolean flags don't have values, they're just true if present on the command line and false if not. There's no need to autocomplete them.

Reply:
This isn't about values, but the flag name. Flags are not autocomplete unless there data.flags is called

Reply:
" Calling this function adds all the flags as autocomplete arguments"
https://bitburner.readthedocs.io/en/latest/netscript/advancedfunctions/autocomplete.html